testsuite/tests/empty-ppc64le4.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+
+# testedfiles: helper.rb
+
+module Yast
```

Comment:
Yes, you are right, I would like to see it for all the tests from now on. Everything is for the first time once...